rosidl_typesupport_connext_cpp/include/rosidl_typesupport_connext_cpp/impl/rosidl_generator_cpp/get_service_type_support_handle.h

```diff
@@ -1,14 +1,14 @@
 #ifndef __rmw__get_service_type_support_handle__h__
 #define __rmw__get_service_type_support_handle__h__
 
-#include "rosidl_typesupport_connext_cpp/ServiceTypeSupport.h"
+#include "rosidl_typesupport_opensplice_cpp/ServiceTypeSupport.h"
```

Comment:
I reverted this particular change since the diff was so small anyways. (it was also moved, but I didn't change that. See: 4288605